query/models.go

```diff
@@ -71,10 +71,13 @@ type QueryModel struct {
 	Table  string
 	Custom map[string]func(ctx context.Context, tx *gorm.DB, val string) (*gorm.DB, error)
 
-	// List of columns that are JSON type.
+	// List of jsonb columns that store a map.
 	// These columns can be addressed using dot notation to access the JSON fields directly
 	// Example: tags.cluster or tags.namespace.
-	JSONColumns []string
+	JSONMapColumns []string
+
+	// List of jsonb columns that store a list.
+	JSONArrayColumns []string
```

Comment:
This doesn't seem to be implemented

Reply:
Tracking that in here - https://github.com/flanksource/duty/issues/1300